packs/actions/change-shape-werecreature-dedication.json

```
@@ -0,0 +1,102 @@
+{
+    "_id": "9zdI8S6ceGIgnAkT",
+    "img": "systems/pf2e/icons/actions/OneAction.webp",
+    "name": "Change Shape (Werecreature Dedication)",
```

```suggestion
    "name": "Change Shape (Werecreature)",
```
File will need renamed, but I think this is a cleaner name for it